### PR TITLE
added MVA vars to CA and DNN evaluation

### PR DESCRIPTION
**Pull request by fmanteca**
- dR = 0.6 in CA -> https://github.com/cms-lpc-llp/run3_llp_analyzer/pull/1/files#diff-4fe5efee2d5980a378e6d41d6a94ddec72236e5914e5529141870c7965dffba5R25
- Added MVA variables 
- DNN evaluation: scripts used to add the DNN score to the analysis ntuples

### CODE REVIEW

File: src/CACluster.cc [hunk 1]
```diff
@@ -259,6 +261,11 @@ void CACluster::clusterProperties()
       // CSCs -> take only one Z value per each station
       
       if(rechit.superlayer > 0){ //hit in DT
+	//save nhits per wheel
+	if(abs(rechit.z) < 126.8){tmpCluster.nDtRechitsWheel0++;}
```
Comment (Christinaw97) [inline src/CACluster.cc]: There's this variable in the big ntuple that's called `dtRechitWheel`, can you use that to assign the wheel instead of Z?

Comment (fmanteca) [reply src/CACluster.cc]: looks like the existing branch stores the wheel which contains the biggest number of hits for each cluster. The new variables I added correspond to the number of hits found in each wheel. 

Several clusters have hits in more than one one wheel:

image

Comment (Christinaw97) [reply src/CACluster.cc]: which existing branch are you referring to? I'm talking about the way you assign the wheels based on the rechit Z position, there's a variable in the big ntuple that gives you the dtRechit wheel: `dtRechitWheel` . So you could change your code to something like this: `if(abs(rechit.dtRechitWheel) == 0){tmpCluster.nDtRechitsWheel0++;}`

Comment (fmanteca) [reply src/CACluster.cc]: Ok, I got confused with what "the big ntuple" meant. I'm updating that

Comment (fmanteca) [reply src/CACluster.cc]: Done

**Pull request merged**